### PR TITLE
Jenkins

### PR DESCRIPTION
**Pull request by sashko**

### CODE REVIEW

File: tools/ci/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+// Copyright (C) 2019 Luxoft Sweden AB
+// SPDX-License-Identifier: MPL-2.0
+
+def image
+
+node("Component") {
+    stage("Clean up the environment") {
+        sh '''
+        rm -rf *
```
Comment (martin-ejdestig) [inline tools/ci/Jenkinsfile]: I missed this in the review for template-dbus-service. Saw it yesterday but did not have access to my GH account on a computer... and now you already merged... but... asking here anyway.

Why is this needed? Just a precaution?

Comment (sashko) [reply tools/ci/Jenkinsfile]: Just a good practice.

Comment (martin-ejdestig) [reply tools/ci/Jenkinsfile]: Hum, OK. One would think that Jenkins would take care of this. If it does, this is just unnecessary noise. If it does not (and is good practice in Jenkins land), we need to do it in a couple of other repos where we do not... ?
